test/typescript/react.tsx

```diff
@@ -0,0 +1,92 @@
+import * as ReactDOM from 'react-dom';
```

Comment:
If you need this file to test typings, you need to use all APIs here. `useSubscription`, DevTools.

In other hand you can remove a lot if unnecessary code from this post (like multiple Trackers).

Reply:
Agree, I'll do it